iFixit/react-commerce
# Add support for pro pricing

fixes #134 

## Changelog

- Show pro pricing to pro users
- Hide the price range filter when a user is pro
- Show pro badge for pro products

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `cominor.com`)
2. Visit the tools product list (`/Store/Tools`)
3. Log in as a pro user in cominor (e.g. `pro_1@ifixit.com` with pwd `iamapro`)
4. Verify that products marked as PRO have a PRO badge (e.g. `Surface Display Debonding Tool`)
5. Verify that pro prices are displayed
6. Verify that the price range filter is not available
